Example plugin from template not building

I'm following the getting started tutorial at [http://docs.ajenti.org/en/latest/dev/intro.html](http://docs.ajenti.org/en/latest/dev/intro.html) but fail to build a basic plugin.

These are the steps I follow:

- Create a new plugin in the current directory:
```
ajenti-dev-multitool --new-plugin "Some plugin name"
```

- Build resources:
```
cd some_plugin_name
ajenti-dev-multitool --build
```

- And start it:
```
sudo ajenti-dev-multitool --run-dev
```

I get this error first for 'ajenti-dev-multitool --build': 
```
yaml.constructor.ConstructorError: could not determine a constructor for the tag 'tag:yaml.org,2002:python/object:aj.plugins.PluginDependency'
  in "./plugin.yml", line 10, column 7
```

After updating 'plugin.yml' from

```
...
dependencies:
    - !!python/object:aj.plugins.PluginDependency {
        plugin_name: core
...
```
To

```
...
dependencies:
    - !PluginDependency {
        plugin_name: core
...
```
Now, I receive this error:

```
INFO Building resources for .
INFO Compiling ./resources/js/module.es
INFO Compiling ./resources/js/routing.es
INFO Compiling ./resources/js/controllers/index.controller.es
ERROR BUILD FAILED
INFO   3 error(s):
ERROR  * ./resources/js/module.es
WARNING b'You have mistakenly installed the `babel` package, which is a no-op in Babel 6.'
WARNING b"Babel's CLI commands have been moved from the `babel` package to the `babel-cli` package."
WARNING b''
WARNING b'    npm uninstall babel'
WARNING b'    npm install --save-dev babel-cli'
WARNING b''
WARNING b'See http://babeljs.io/docs/usage/cli/ for setup instructions.'
ERROR  * ./resources/js/controllers/index.controller.es
WARNING b'You have mistakenly installed the `babel` package, which is a no-op in Babel 6.'
WARNING b"Babel's CLI commands have been moved from the `babel` package to the `babel-cli` package."
WARNING b''
WARNING b'    npm uninstall babel'
WARNING b'    npm install --save-dev babel-cli'
WARNING b''
WARNING b'See http://babeljs.io/docs/usage/cli/ for setup instructions.'
ERROR  * ./resources/js/routing.es
WARNING b'You have mistakenly installed the `babel` package, which is a no-op in Babel 6.'
WARNING b"Babel's CLI commands have been moved from the `babel` package to the `babel-cli` package."
WARNING b''
WARNING b'    npm uninstall babel'
WARNING b'    npm install --save-dev babel-cli'
WARNING b''
WARNING b'See http://babeljs.io/docs/usage/cli/ for setup instructions.'
```
Anyone know what's going wrong or can give me a link to a working demo plugin?

Thanks in advance!

--- a/plugin.yml
+++ b/plugin.yml
@@ -7,16 +7,16 @@ title: 'my_plugin'
 icon: question
 # dependency on Core is required for UI plugins
 dependencies:
-    - !!python/object:aj.plugins.PluginDependency {
+    - !PluginDependency {
         plugin_name: core
     }
-    # - !!python/object:aj.plugins.OptionalPluginDependency {
+    # - !OptionalPluginDependency {
     #   plugin_name: my_other_plugin
     # }
-    # - !!python/object:aj.plugins.ModuleDependency {
+    # - !ModuleDependency {
     #   module_name: requests
     # }
-    # - !!python/object:aj.plugins.BinaryDependency {
+    # - !BinaryDependency {
     #   binary_name: curl
     # }
 resources:

